sdk/keyvault/Azure.Security.KeyVault.Certificates/src/SubjectAlternativeNames.cs

```diff
@@ -83,10 +92,12 @@ public static SubjectAlternativeNames FromEmail(IEnumerable<string> names)
         }
 
         /// <summary>
-        /// Creates a collection of unique principal name (UPN) subject alternative names (SANs)
+        /// Creates a collection of unique principal names (UPNs) subject alternative names (SANs).
```

Comment:
It's confusing to have a plural modifier of a plural subject.  Can we change this?